### DISCUSSION TITLE
Difference between free & pay version

### DISCUSSION THREAD

**Discussion by GerdRuetten** [LeoKlaus/Paperparrot#127] [General]
Hi,

first, many thanks for this great app!

Have you a tableview to show the difference between free and pay version from this app?

Best regards

Gerd

**Comment by LeoKlaus** [comment]
Hi Gerd,
all features are available in the free version of Paperparrot. The only limit is that it will only display up to six documents or one search result.
If you came from Caschys Blog, you might be able to grab one of the codes I posted to unlock the full version for free ;)
